pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -382,4 +400,60 @@ private void addExtraFiles(File m2Repo) {
                     strictLicenseCheck);
         }
     }
+
+    private void generateOfflinerManifest() {
```

Comment:
I know we haven't documented all the other strategies yet, but could you please write a Javadoc that explains what the offliner manifest does please? That would help us writing our documentation for the future :)

Reply:
Done

Reply:
It looks like you need to move `close()` to the bottom and make sure that the javadoc is on the correct method here.